### ISSUE TITLE
Remove .DS_Store files in downloaded zip files

### ISSUE THREAD

**Issue by zhangsu** [altryne/extensionizr#30]
.DS_Store files, which are specific to OS X file manager, are found in the downloaded zip files.

**Comment by arsenicraghav** [comment]
should we add the .DS_Store file to .gitignore file ?

**Comment by darthwalsh** [comment]
@arsenicraghav No, it's an [artifact of zip files created on a mac](https://superuser.com/a/757596/282374). Just delete it.
